ui-components/src/navigation/ProgressNav.tsx

```diff
@@ -19,7 +19,9 @@ const ProgressNavItem = (props: {
 
   return (
     <li className={`progress-nav__item ${bgColor}`}>
-      <a href="#">{props.label}</a>
+      <a aria-disabled={bgColor ? "true" : "false"} href="#">
```

Comment:
Do we want to explicitly check whether bgColor is disabled, since it can be active too? It would also make it easier for people reading the code to understand why we're checking bgColor to determine the value of aria disabled. Would be even better if we define a constant variable for the string "is-disabled" and "is-active", so that all instances of the string will be updated together in the future, preventing bugs. 

```suggestion
const IS_LINK_DISABLED = "is-disabled"; 
//...
      <a aria-disabled={bgColor == IS_LINK_DISABLED} href="#">
```

Reply:
Refreshed in latest commit.